app/src/main/java/com/android/brewr/ui/explore/Explore.kt

```diff
@@ -15,29 +15,50 @@ import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.unit.dp
-import com.android.brewr.model.coffee.Coffee
-import com.android.brewr.ui.explore.CoffeeInformationScreen
-import com.android.brewr.ui.explore.MapScreen
+import com.android.brewr.model.coffee.CoffeesViewModel
 import kotlinx.coroutines.launch
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun ExploreScreen(coffees: List<Coffee>) {
+fun ExploreScreen(coffeesViewModel: CoffeesViewModel) {
```

Comment:
Now I see why you have added a CoffeesViewModel ! I think it's way better than just a list of Coffee. Great idea !

Reply:
Thanks!!